tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+
+4. To connect to the Local Fabric gateway using this identity, you simply click `Local Fabric` under the Fabric Gateways panel and select the identity you wish to connect with (Tesla in our case).
+> Command Palette alternative: `Create Identity (register and enroll)`
+</details>
+
+<details>
+<summary><b>2. Using our new identity</b></summary>
+
+Now that we've created our very own identities, we can implement attribute-based action control. To do so, follow the steps below:
+
+1. Click the `Explorer` icon in the left sidebar (it will probably be at the top and will look like a folder) and navigate to the `car-contract.ts` file in the `src` folder under the contract directory.
+
+2. To deny Jack from carrying out the `createCar()` transaction, we need to tweak the generated smart contract slightly. We can change the body of the function to look like this: 
+<details>
+<summary> TypeScript </summary>
+
+```
+const identity: ClientIdentity = ctx.clientIdentity;
+// Check if the identity has the 'manufacturer' attribute set to 'true'
+const checkAttr: boolean = identity.assertAttributeValue('manufacturer', 'true');
+if (checkAttr) {
+    const exists = await this.carExists(ctx, carId);
+    if (exists) {
+        throw new Error(`The car ${carId} already exists`);
+    }
+    const car = new Car();
+    car.value = value;
+    const buffer = Buffer.from(JSON.stringify(car));
+    await ctx.stub.putState(carId, buffer);
+} else {
+    throw new Error('You must be a manufacturer to carry out this transaction!');
+}
+```
+
+</details>
+
+The above simply checks if the identity trying to access the `createCar()` function (Jack in our case) has the attribute `manufacturer` set to `true`. If it does, allow the transaction to be executed. If the identity does not have the attribute `manufacturer` (or it isn't set to true), do not let the identity carry out the transaction and show an error message.
+
+3. To test that this works, we can connect to the gateway by pressing `Local Fabric` under the `FABRIC GATEWAYS` panel. You will now be asked which identity you wish to connect with - choose `Jack`. 
+> Command Palette alternative: `Connect Via Gateway`
+
+4. To execute a function, navigate to `mychannel`, `yourContractNameHere` under the `FABRIC GATEWAYS` panel. Here, you will see a list of all the transactions you can carry out. For the scope of this tutorial, we will be submitting the `createCar()` transaction (right-click on the transaction and `Submit Transaction`). Say we want to create a car with the arguments `["001", "Model X"]`, it will show the error message `Error: You must be a manufacturer to carry out this transaction!` because Jack is not a manufacturer!
```

Comment:
you tell them to update the contract but you don't tell them to either instantiate or upgrade it!

Reply:
Fixed - added steps on how to install + instantiate (used from previous tutorial)